<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="code">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Szyfruj</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="decode">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>250</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Odszyfruj</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Szyfr ADFGVX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textFrom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>471</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textTo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>471</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_run">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>500</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textKey">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelKey">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Klucz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
